src/main/java/luke/data/TaskList.java

```diff
@@ -0,0 +1,89 @@
+package luke.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
+
+import luke.data.tasks.Task;
+import luke.storage.Storable;
+
+/**
+ * Implements a list for storing tasks.
+ * It implements Storable to store the data in a file using StorageFile.
+ */
+public class TaskList implements Storable {
+    private List<Task> taskList;
+
+    /**
+     * Constructs an empty task list.
+     */
+    public TaskList() {
+        taskList = new ArrayList<>();
+    }
+
```

Comment:
Maybe you could add a method header for getData() method. 👍

Reply:
Good suggestion! I will add that in in the next commit :)